catroid/src/main/java/org/catrobat/catroid/content/ActionFactory.java

```diff
@@ -1450,7 +1450,7 @@ public Action createReadVariableFromFileAction(Sprite sprite, SequenceAction seq
 
 	public Action createWriteListOnDeviceAction(UserList userList) {
 		WriteListOnDeviceAction action = Actions.action(WriteListOnDeviceAction.class);
-		action.setUserList(userList);
+		action.userList = userList;
```

Comment:
Please leave it here like it was

Reply:
as you suggested, I've removed the setUserList function, so its kinda necessary here😅